front/src/views/Schedule.vue

```diff
@@ -47,7 +47,7 @@
                             persistent
                             @save="save(props.item)"
                         >
-                            <div>{{ props.item.theme }}</div>
+                            <div>{{ props.item.theme || '✏️' }}</div>
```

Comment:
maybe we should have always a pen to show it's editable ?

Reply:
Can we test a first version like that ?